modules/snooker/src/test/ts/browser/UnmergeOperationsTest.ts

```diff
@@ -109,4 +109,109 @@ UnitTest.test('UnmergeOperationsTest', function () {
       { section: 1, row: 0, column: 0 }
     ]
   );
+
+  Assertions.checkUnmerge(
+    '<table>' +
+      '<tbody>' +
+        '<tr>' +
+          '<th scope="row">A1</th>' +
+          '<td>B1</td>' +
+          '<td>C1</td>' +
+          '<td>D1</td>' +
+        '</tr>' +
+        '<tr>' +
+          '<th scope="row">?</th>' +
+          '<td>B2</td>' +
+          '<td>C2</td>' +
+          '<td>D2</td>' +
+        '</tr>' +
+        '<tr>' +
+          '<th scope="row">?</th>' +
+          '<td>B3</td>' +
+          '<td>C3</td>' +
+          '<td>D3</td>' +
+        '</tr>' +
+      '</tbody>' +
+    '</table>',
+
+    '<table>' +
+      '<tbody>' +
+        '<tr>' +
+          '<th rowspan="3" scope="rowgroup">A1</th>' +
+          '<td>B1</td>' +
+          '<td>C1</td>' +
+          '<td>D1</td>' +
+        '</tr>' +
+        '<tr>' +
+          '<td>B2</td>' +
+          '<td>C2</td>' +
+          '<td>D2</td>' +
+        '</tr>' +
+        '<tr>' +
+          '<td>B3</td>' +
+          '<td>C3</td>' +
+          '<td>D3</td>' +
+        '</tr>' +
+      '</tbody>' +
+    '</table>',
+    [
+      {
+        section: 0,
+        row: 0,
+        column: 0
+      }
+    ]
+  );
+
+  Assertions.checkUnmerge(
+    '<table>' +
+      '<tbody>' +
+        '<tr>' +
+          '<th scope="col">A1</th>' +
+          '<th scope="col">?</th>' +
+          '<th scope="col">?</th>' +
+          '<th scope="col">?</th>' +
+        '</tr>' +
+        '<tr>' +
+          '<td>A2</td>' +
+          '<td>B2</td>' +
+          '<td>C2</td>' +
+          '<td>D2</td>' +
+        '</tr>' +
+        '<tr>' +
+          '<td>A3</td>' +
+          '<td>B3</td>' +
+          '<td>C3</td>' +
+          '<td>D3</td>' +
+        '</tr>' +
+      '</tbody>' +
+    '</table>',
+
+    '<table>' +
+      '<tbody>' +
+        '<tr>' +
+          '<th colspan="4" scope="colgroup">A1</th>' +
+        '</tr>' +
+        '<tr>' +
+          '<td>A2</td>' +
+          '<td>B2</td>' +
+          '<td>C2</td>' +
+          '<td>D2</td>' +
+        '</tr>' +
+        '<tr>' +
+          '<td>A3</td>' +
+          '<td>B3</td>' +
+          '<td>C3</td>' +
+          '<td>D3</td>' +
+        '</tr>' +
+      '</tbody>' +
+    '</table>',
+    [
+      {
+        section: 0,
+        row: 0,
+        column: 0
+      }
+    ]
+  );
```

Comment:
What about
- Unmerging a cell where the span and the scope don't match (eg rowspan + colgroup / colspan + rowgroup)
- Unmerging cells that have both rowspan and colspan, as well as some form of scope attribute

Reply:
Not looked at. We only look at the scope attribute. In the first case it'd result in row/col depending on the scope, same for the second.

Reply:
Is it worth making a Jira to look into this kind of thing?

Reply:
Neil has we need a test case for the second point at least. As you could have something like this:
```
<table>
  <tbody>
    <tr><td scope="colgroup" colspan="3">a</td></tr>
    <tr><td scope="rowgroup rowspan="2">b</td><td>c</td><td>d</td></tr>
    <tr><td>e</td><td>f</td></tr>
  </tbody>
</table>
```
What happens if you do a multi-cell selection and click unmerge.